src/views/Order.vue

```diff
@@ -283,10 +286,10 @@ export default defineComponent({
         this.customerAddress = this.order.shipGroup.shipTo?.postalAddress ? this.order.shipGroup.shipTo.postalAddress : {}
         await this.getPickupStores();
         this.fetchOrderFacilityChangeHistory()
-        if(!this.nearbyStores.length) {
+        if(!this.nearbyStores.length && this.hasMultipleItems()) {
```

Comment:
Make hasMultipleItems as well a data property, instead of having it as a method.